src/pages/ExpiredCardScreen.tsx

```diff
@@ -1,19 +1,33 @@
-import React from 'react'
+import React, { useEffect } from 'react'
 import Prose from '../components/Prose'
 import Main, { MainChild } from '../components/Main'
 import Screen from '../components/Screen'
+import { PartialUserSettings } from '../config/types'
 
-const ExpiredCardScreen = () => (
-  <Screen>
-    <Main>
-      <MainChild center>
-        <Prose textCenter>
-          <h1>Expired Card</h1>
-          <p>Please see poll worker for assistance.</p>
-        </Prose>
-      </MainChild>
-    </Main>
-  </Screen>
-)
+interface Props {
+  setUserSettings: (partial: PartialUserSettings) => void
+}
+
+const ExpiredCardScreen = ({ setUserSettings }: Props) => {
+  useEffect(() => {
+    setUserSettings({ textSize: 3 })
+    return () => {
+      setUserSettings({ textSize: 1 })
+    }
+  }, [setUserSettings])
```

Comment:
This is a little odd. I guess it doesn't matter that you've lost the voter's settings because they can't continue anyway. Should the default of 1 come from a constant?

Reply:
will do.